### PR TITLE
Generate subscripition client of version 2021-01-01

### PR DESCRIPTION
**Pull request by msJinLei**
This PR generates subscripition client of version 2021-01-01
It has been tested with azure-powershell

### CODE REVIEW

File: src/ResourceManager/Version2021_01_01/Customized/SubscriptionExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Linq;
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Management.ResourceManager.Version2021_01_01.Models.Utilities
```
Comment (isra-fel) [inline src/ResourceManager/Version2021_01_01/Customized/SubscriptionExtensions.cs:19]: I think the namespace should be `Microsoft.Azure.Commands.ResourceManager.Version2021_01_01.Customized`, just like `SubscriptionClientExtension.cs`

File: src/ResourceManager/Version2021_01_01/Customized/TenantIdDescriptionExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Linq;
+using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.Commands.ResourceManager.Common;
+
+namespace Microsoft.Azure.Management.ResourceManager.Version2021_01_01.Models.Utilities
```
Comment (isra-fel) [inline src/ResourceManager/Version2021_01_01/Customized/TenantIdDescriptionExtensions.cs:20]: Here too

File: src/ResourceManager/Version2021_01_01/readme.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+namespace: Microsoft.Azure.Management.ResourceManager.Version2021_01_01
+
+#directive:
+# - remove-operation:
+#   - Operations_List
+#   - Subscriptions_ListLocations
+#   - Subscriptions_Get
+#   - Subscriptions_List
+#   - Tenants_List
+#   - checkResourceName
```
Comment (isra-fel) [inline src/ResourceManager/Version2021_01_01/readme.md:40]: Does this mean we want to keep all the operations?

**Review by isra-fel — COMMENTED** [review]
Have you tested this new version?

File: src/ResourceManager/Version2021_01_01/readme.md (hunk 3 above)
Comment (msJinLei) [reply src/ResourceManager/Version2021_01_01/readme.md:40]: yes, the operations are not so many. I would like to keep them all.

File: src/ResourceManager/Version2021_01_01/Customized/SubscriptionExtensions.cs (hunk 1 above)
Comment (msJinLei) [reply src/ResourceManager/Version2021_01_01/Customized/SubscriptionExtensions.cs:19]: To keep the consistent with the existing codes, we prefer to use Microsoft.Azure.Management.ResourceManager.Version2021_01_01.Models.Utilities.

**Review by isra-fel — APPROVED** [review]
LGTM

File: src/ResourceManager/Version2021_01_01/readme.md (hunk 3 above)
Comment (msJinLei) [reply src/ResourceManager/Version2021_01_01/readme.md:40]: check subscription client with previous version and find all the operations are kept.

**Pull request merged**